oio/account/server.py

```diff
@@ -23,6 +23,21 @@
 from oio.common.wsgi import WerkzeugApp
 
 
+def access_log(func):
+    from functools import wraps
+
+    @wraps(func)
+    def _access_log_wrapper(self, *args, **kwargs):
+        from time import time
```

Comment:
No need to delay this import statement. Also, you may want to import `oio.common.utils.monotonic_time` from 4.x branch.